community/libxkbcommon/build

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh -e
+
+export DESTDIR="$1"
+
+meson \
+    --prefix=/usr \
+    --sysconfdir=/usr \
```

Comment:
This and `--mandir` may not need adding if these values match the defaults.

Reply:
you're right. fixed!